core/src/main/java/com/sequenceiq/cloudbreak/service/LoadBalancerConfigService.java

```diff
@@ -389,6 +434,31 @@ private boolean isEndpointGatewayEnabled(EnvironmentNetworkResponse network) {
         return Optional.ofNullable(knoxTargetGroup);
     }
 
+    private Optional<TargetGroup> setupOozieHATargetGroup(Stack stack, boolean dryRun) {
+        TargetGroup oozieTargetGroup = null;
+        Set<String> oozieGroupNames = getOozieGroups(stack);
+        InstanceGroup oozieInstanceGroup = stack.getInstanceGroups().stream()
+            .filter(ig -> oozieGroupNames.contains(ig.getGroupName()))
+            .filter(ig -> ig.getNodeCount() > 1)
```

Comment:
so this can't change during an upscale, and LB only needed during HA, is this right?

Reply:
Yup, there is also other enhancements in progress like specifying min node count to 2 so that the customer can not also downscale or start with 0 or 1 nodes.